dags/utils/slack_operator.py

```diff
@@ -1,11 +1,60 @@
 from airflow.hooks.base_hook import BaseHook
 from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
 
+
 # This is the Conn Id that we set when creating the connection in the Airflow dashboard
 # in Admin > Connections.
 SLACK_CONN_ID = "slack"
 
 
+def format_schedule(schedule_interval):
+    """
+    Convert a schedule interval to a human-readable format.
+    This function handles different types of schedule intervals:
+    - None: Returns "None" as the description.
+    - str: Assumes the string is a cron expression and tries to convert it to a
+        human-readable format. If conversion fails, it returns the cron
+        expression as is.
+    - datetime.timedelta: Converts the timedelta to a human-readable string,
+        breaking it down into days, hours, minutes, and seconds.
+    - Other types: Converts the interval to a string representation.
+    Args:
+        schedule_interval: The schedule interval to format. It can be None, a
+        string (cron expression), or a datetime.timedelta.
+    Returns:
+        str: A human-readable description of the schedule interval.
+    """
+
+    from cron_descriptor import get_description
+
+    if schedule_interval is None:
+        schedule_description = "None"
+    elif isinstance(schedule_interval, str):
+        try:
+            schedule_description = get_description(schedule_interval)
+        except Exception:
+            schedule_description = f"Cron expression: {schedule_interval}"
+    elif isinstance(schedule_interval, datetime.timedelta):
```

Comment:
i ran into this error when I tested a schedule that is a timedelta object. just needs to be imported
```
File "/opt/airflow/dags/utils/slack_operator.py", line 37, in format_schedule
    elif isinstance(schedule_interval, datetime.timedelta):
                                       ^^^^^^^^
NameError: name 'datetime' is not defined. Did you forget to import 'datetime'
```

Reply:
Thank you Mike https://github.com/cityofaustin/atd-airflow/pull/264/commits/7d6f4192bbfe736b1d9995744c2b421f40a9bb37